src/k6helper.ts

```diff
@@ -290,3 +297,37 @@ export function getInstalledK6Version(): string {
     return ''
   }
 }
+
+/**
+ * Compares two semantic version strings and checks if version1 is at least (greater than or equal to) version2.
+ *
+ * @param {string} version1 - The first version string (e.g., "0.56.0")
+ * @param {string} version2 - The second version string (e.g., "0.55.0")
+ * @returns {boolean} True if version1 is at least version2, false otherwise
+ */
+export function isVersionAtLeast(
```

Comment:
Why not use something like  - https://www.npmjs.com/package/semver

Reply:
Wasn't sure you would want me to add a dependency for a small function like this, but happy to use `semver` if that's preferred. Gives some flexibility for the future too (they include more utility functions to compare versions). Will update and push